ansible_playbook/install_log2ram.yml

```diff
@@ -1,32 +1,75 @@
---- 
+#!/usr/bin/env ansible-playbook
+---
+# Configure Raspberry Pi to log to RAM, with occasional SD card sync
+# to reduce SD card writes
+# Usage: ansible-playbook -e 'log2ram_size=80M' install_log2ram.yml
+
 - hosts: all
+  gather_facts: true
+  gather_subset: min
   become: true
+
+  vars:
+    apt_keyring_file: azlux.fr.log2ram.gpg
+    apt_sources_file: log2ram.list
+
+    log2ram_size: 48M
+    log2ram_use_rsync: true
+    log2ram_path_disk: /var/log
+    log2ram_use_z2lr: true
+    log2ram_comp_alg: lz4
+    log2ram_log_disk_size: 100M
+
   tasks:
-  - name: Add apt-key for new repository
-    apt_key:
-      keyserver: keyserver.ubuntu.com
-      id: CA548A0A0312D8E6
-  - name: Add log2ram to sources.list
-    apt_repository:
-      repo: deb http://packages.azlux.fr/debian/ buster main
-      state: present
-  - name: update pkg database
-    apt:
-      upgrade: yes
-      update_cache: yes
-  - name: Install log2ram
-    apt:
-      pkg:
-      - log2ram
-
-  - name: modify config
-    lineinfile:
-      path: /etc/log2ram.conf
-      regexp: '^SIZE=(.*)$'
-      line: 'SIZE=80M'
-      backrefs: yes
-
-  - name: Start log2ram service
-    systemd:
-      state: started
-      name: log2ram
+    - name: "Add /etc/apt/sources.list.d/{{ apt_sources_file }}"
+      ansible.builtin.copy:
+        content: "deb http://packages.azlux.fr/debian/ {{ ansible_facts['distribution_release'] }} main\n"
+        dest: /etc/apt/sources.list.d/{{ apt_sources_file }}
+        mode: 0644
+        owner: root
+      register: source_list
+
+    - name: Import apt key
+      ansible.builtin.apt_key:
+        url: https://azlux.fr/repo.gpg.key
+        id: 98B824A5FA7D3A10FDB225B7CA548A0A0312D8E6
+        keyring: /etc/apt/trusted.gpg.d/{{ apt_keyring_file }}
+      register: dl_key
+
+    - name: Remove tilde backup file {{ apt_keyring_file }}~
+      ansible.builtin.file:
+        path: /etc/apt/trusted.gpg.d/{{ apt_keyring_file }}~
+        state: absent
+
+    - name: update apt cache
+      ansible.builtin.apt:
+        update_cache: true
+      when: dl_key is changed or source_list is changed
+
+    - name: Install log2ram
+      ansible.builtin.apt:
+        pkg:
+          - log2ram
+          - rsync
+      register: pkg
+
+    - name: Set config options
+      ansible.builtin.lineinfile:
+        path: /etc/log2ram.conf
+        regexp: "{{ item.regexp }}"
+        line: "{{ item.line }}"
+        backrefs: true
+      loop:
+        - {regexp: '^SIZE=(.*)$', line: 'SIZE={{ log2ram_size }}'}
+        - {regexp: '^USE_RSYNC=(.*)$', line: 'USE_RSYNC={{ log2ram_use_rsync|lower }}'}
```

Comment:
There are no config option for rsync anymore.
https://github.com/azlux/log2ram/blob/master/log2ram#L50 It's a simple check now.

Reply:
Thanks for the heads-up, I'll update the MR.